$ cp plasma6.nix.orig plasma6.nix && patch --backup --verbose < plasma6.patch
Index: plasma6.nix
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/plasma6.nix b/plasma6.nix
--- a/plasma6.nix
+++ b/plasma6.nix
@@ -180,8 +180,7 @@
           discover
         ];
       in
-      requiredPackages
-      ++ utils.removePackagesByName optionalPackages config.environment.plasma6.excludePackages
+      utils.removePackagesByName (requiredPackages ++ optionalPackages) config.environment.plasma6.excludePackages
       ++ lib.optionals config.services.desktopManager.plasma6.enableQt5Integration [
         breeze.qt5
         plasma-integration.qt5
